test/helpers/pagelets_helper_test.rb

```diff
@@ -1,17 +1,17 @@
 require 'test_helper'
+require 'statics_isolator_helper'
 
 class PageletsHelperTest < ActionView::TestCase
+  include Isolator
   include PageletsHelper
 
+  isolate_config Pagelets::Manager
```

Comment:
I am adding the pagelets in the initializer. Many tests are relying on the fact that puppet tabs are in place without extra setup, which sounds perfect to me (they should be there in prod). The problem here is that pagelet tests are clearing a global registry.
The same problem will happen when I'll move puppet models to facets - when the registry will be cleared in facet tests, it will loose facets registered in the initializers.